src/core_modules/capture-core/metaDataStoreLoaders/IOUtils/quickStore.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import { query } from './query';
+import { getContext } from '../context';
+
+import type { ApiQueryExtended, StoreName, QuickStoreOptions } from './IOUtils.types';
+
+/**
+ * Queries the Api, converts the result using the callback and then stores it all in the selected storage.
+ *
+ * @param {ApiQueryExtended} querySpecification: the specification of the api query to run
+ * @param {string} storeName: the name of the store supplied to the storageController
+ * @param {Options} options
+ */
+export const quickStore = async (
+    querySpecification: ApiQueryExtended,
+    storeName: StoreName,
+    {
+        onConvert,
```

Comment:
It looks to me that `onCovert` is not an optional parameter. Is it really optional?

Also can we discuss the name `onConvert`? Maybe something between the lines of `convertQueryResponse` is more descriptive?

Reply:
> It looks to me that onCovert is not an optional parameter. Is it really optional?

Good catch. Thanks. Fixing.

> Also can we discuss the name onConvert? Maybe something between the lines of convertQueryResponse is more descriptive?

Using your suggestion 👍

Reply:
well I am happy you are willing to change it, and thanks for that. One point though do we need the `on` on`onConvertQueryResponse`? Isnt it simply converting the response?

Reply:
Thanks for bringing this to my attention. Bad habit of mine right there, this is simply requesting a processing so will skip the on part.